kitsune/questions/jinja2/questions/includes/aaq_macros.html

```diff
@@ -83,7 +83,23 @@ <h3 class="card--title">
   </ul>
 {%- endmacro %}
 
-{% macro explore_solutions(product, search_box, featured, topics) -%}
+{% macro aaq_widget(key, user, location="aaq") %}
```

Comment:
You don't need the user here. You can check the `request.user` instead to get auth status.